libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -115,12 +115,12 @@ AP_BattMonitor::init()
 #if HAL_BATTMON_SMBUS_ENABLE
             case AP_BattMonitor_Params::BattMonitor_TYPE_SOLO:
                 drivers[instance] = new AP_BattMonitor_SMBus_Solo(*this, state[instance], _params[instance],
-                                                                  hal.i2c_mgr->get_device(AP_BATTMONITOR_SMBUS_BUS_INTERNAL, AP_BATTMONITOR_SMBUS_I2C_ADDR,
```

Comment:
this will break existing setups. See this PR for correct fix for bus default:
https://github.com/ArduPilot/ardupilot/pull/12979

Reply:
Great, thanks.  I've included the defaulting of the Solo and Maxel battery I2C bus in a follow-up commit.